src/Stratis.Bitcoin.Features.Consensus/ConsensusLoop.cs

```diff
@@ -471,7 +471,8 @@ public void ValidateBlock(ContextInformation context)
                 // Build the next block in the chain of headers. The chain header is most likely already created by
                 // one of the peers so after we create a new chained block (mainly for validation)
                 // we ask the chain headers for its version (also to prevent memory leaks).
-                context.BlockValidationContext.ChainedBlock = new ChainedBlock(context.BlockValidationContext.Block.Header, context.BlockValidationContext.Block.Header.GetHash(), this.Tip);
+                context.BlockValidationContext.ChainedBlock = new ChainedBlock(context.BlockValidationContext.Block.Header, 
+                    context.BlockValidationContext.Block.Header.GetHash(), this.Tip);
```

Comment:
You forgot to pass `consensus.NetworkOptions` to the `new ChainedBlock(` in GetHash()

Reply:
Fixed to pass this.connectionManager.Network.(Consesnsus.)NetworkOptions.